server/push_test.go

```diff
@@ -1457,7 +1461,7 @@ func TestPush_ReuseIntmidWithDiffExtmid(t *testing.T) {
 	defer goleak.VerifyNone(t, common.IgnoreRoutines()...)
 	core.JsonPlaylistQuitTimeout = 0 * time.Second
 
-	reader := strings.NewReader("InsteadOf.TS")
+	reader := strings.NewReader("setupServerWithCancel()")
```

Comment:
Just out of curiosity, Is there any special reason to put this string?

Reply:
I was on a copy-pasting spree apparently, will change it back - it's just a placeholder, though